app/controllers/api/v1/organizations_controller.rb

```diff
@@ -0,0 +1,10 @@
+class Api::V1::OrganizationsController < ApplicationController
+
+  #GET /organization/public
+  def show
+    @organization = Organization.all
+
+    render :json => @organizations.to_json(:only => [:name, :image, :phone, :address])
```

Comment:
I recommend exploring the [jsonapi serializer gem](https://github.com/jsonapi-serializer/jsonapi-serializer) to serialize output.